lib/gollum/app.rb

```diff
@@ -77,6 +80,8 @@ class App < Sinatra::Base
       @critic_markup = settings.wiki_options[:critic_markup]
       @redirects_enabled = settings.wiki_options.fetch(:redirects_enabled, true)
       @per_page_uploads = settings.wiki_options[:per_page_uploads]
+      
+      @wiki_title = settings.wiki_options.fetch(:title, 'Gollum Wiki')
```

Comment:
There is no switch for this in `bin/gollum`, but someone could use a `config.rb` to set `Precious::App.wiki_options[:title] = 'My Title`. That ok, or should we add a `--title` switch? Downside would be: the RSS feed is currently the only place where the title makes a difference.

Reply:
> but someone could use a config.rb to set Precious::App.wiki_options[:title] = 'My Title'

Perfectly acceptable!

Reply:
Perfectly acceptable!